### PR TITLE
Setup DB & Add entry point /api/stations

### PR DESCRIPTION
**Pull request by kumilange**
Sorry, guys. Wanted to separate PRs but we don't seem to have time to wait for each to be approved so I put them all together.

1. create table air_delivery
2. populate station location data
3. connect with DB
4. add entry point `/api/stations`
5. fix lint errors (Still have some errors. working on it)

### CODE REVIEW

File: server/bin/www [hunk 1]
```diff
@@ -1,25 +1,32 @@
 #!/usr/bin/env node
 
 /**
+ * Run database migrations
+ */
+// TODO add 'run database migrations'
+// console.log('Running migrations...');
+// db.migrate.latest();
+
```
Comment (billfeng) [inline server/bin/www]: If we don't need this anymore, please delete it :) don't leave comments in the code on master branch.

File: server/bin/www [hunk 2]
```diff
@@ -82,8 +89,8 @@ function onError(error) {
  */
 
 function onListening() {
-  var addr = server.address();
-  var bind = typeof addr === 'string'
+  const addr = server.address();
+  const bind = typeof addr === 'string'
```
Comment (billfeng) [inline server/bin/www:86]: Thanks for fixing these!

File: server/routes/api.js [hunk 3]
```diff
@@ -2,9 +2,16 @@ const express = require('express');
 
 const router = express.Router();
 
-/* GET users listing. */
-router.get('/', (req, res) => {
-  res.json({ page: 'Api' });
-});
+module.exports = (services) => {
+  /* GET users listing. */
```
Comment (billfeng) [inline server/routes/api.js]: This doesn't look like user listing anymore, so please fix or remove this comment.

**Review by billfeng — CHANGES_REQUESTED** [review]
A few things to correct + linter. Otherwise great job! :)

File: server/bin/www (hunk 1 above)
Comment (kumilange) [reply server/bin/www]: @bill-feng Thanks for noticing! Understand, removed it!:)

File: server/routes/api.js (hunk 3 above)
Comment (kumilange) [reply server/routes/api.js]: @bill-feng Thanks! I fixed the comment!:)

**Review by pegaheinaliyan — APPROVED** [review]

**Review by billfeng — APPROVED** [review]

**Pull request merged**